pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/Lookup.java

```diff
@@ -41,6 +42,14 @@
      */
     CompletableFuture<String> lookupTopicAsync(String topic);
 
+    /**
+     * Lookup a partitioned topic.
+     *
+     * @param topic
+     * @return the broker URL that serves the topic
+     */
```

Comment:
This method should return a CompletableFuture as the method above

Reply:
This is not a `async` method. It invokes Lookup#lookupTopicAsync internal.

Reply:
I meant that we should provide an async method.
Why not?

Reply:
For the uniform of method name, we should add a `aync` method for `lookupPartitionedTopic`. 
But other method without `async`  juset  invoke the relative `async` method. 
e.g.
`getBundleRange` just invoke ` getBundleRangeAsync(topic).get(this.readTimeoutMs, TimeUnit.MILLISECONDS)`
But, `lookupPartitionedTopic` internal invokes `topics.getPartitionedTopicMetadataAsync(topic).get(this.readTimeoutMs, TimeUnit.MILLISECONDS)` and 
` lookupTopicAsync(partitionTopicName).get(readTimeoutMs, TimeUnit.MILLISECONDS)`. 
So it's hard to  tell which is better.

Reply:
Personally I would add lookupPartitionedTopicAsync

If the user wants to block it can do it.
If we provide only the blocking method it is not possible to turn it into non-blocking

Reply:
Personally I would add lookupPartitionedTopicAsync

If the user wants to block it can do it.
If we provide only the blocking method it is not possible to turn it into non-blocking

Reply:
Thanks for your suggenstions @eolivelli .